File: 1873b.py
t = int(input())
answer = []
for i in range(t):
    n = int(input())
    a = list(map(int, input().split()))
    a.sort()
    a[0] += 1
    ans = 1
    for i in range(len(a)):
        ans *= a[i]
    answer.append(ans)
print(*answer, sep='\n')

File: 1873b.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int t;
    cin >> t;
    while(t--){
        int n;
        cin >> n;
        vector<int> v(n);
        for (int i=0; i<n; i++){
            cin >> v[i];
        }
        sort(v.begin(), v.end());
        v[0] = v[0]+1;
        int answer=1;
        for (int x : v){
            answer *= x;
        }
        cout << answer << endl;
    }
    
    return 0;
}